Card/build.gradle

```diff
@@ -42,4 +43,18 @@ dependencies {
     testImplementation 'junit:junit:4.+'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+
+    detektPlugins "io.gitlab.arturbosch.detekt:detekt-formatting:1.18.0-RC2"
```

Comment:
@sarahkoop and I chatted a little bit about importing RC/beta/alpha releases. We came to the conclusion that since we're not shipping detekt in the SDK, we're probably safe using the RC release here. What does everyone else think?

Reply:
yeah it seems fine to me